torch/distributed/checkpoint/default_planner.py

```diff
@@ -64,6 +64,7 @@
     "create_default_global_save_plan",
 ]
 
+CURRENT_DCP_VERSION: float = 1.0
```

Comment:
```suggestion
CURRENT_DCP_VERSION: Final[Literal[float]] = 1.0
```
Types it as close to a constant as you can get in Python.